.vscodeignore

```diff
@@ -1,4 +1,3 @@
-.vscode/**
```

Comment:
The files under `.vscode` are not needed at runtime, we should ignore them, see https://code.visualstudio.com/api/working-with-extensions/publishing-extension#.vscodeignore